plugins/globalVariables.ts

```diff
@@ -14,6 +14,11 @@ useOperators(OperatorType.ACCUMULATOR, {
   $push,
   $avg,
 } as OperatorMap)
+
+/**
+ * skipcq
+ * no-floating-promises: ignoreIIFE
+ */
 ;(async () => {
   await enableExtension()
 })()
```

Comment:
For more details, see here: [ignoreIIFE](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-floating-promises.md#ignoreiife).

Reply:
Instead of hiding the error, isn't there another solution?
I would see something more like
```js
export default async () => {
  await enableExtension()
}
```

Reply:
I will try it, thank you!